src/pages/AddItem/styles.css

```diff
@@ -1,8 +1,42 @@
+.add-container {
+  border-radius: 25px;
+  height: 90vh;
+  width: 500px;
+}
+
 .add-form {
+  text-align: left;
   align-items: center;
   flex-direction: column;
   justify-content: center;
   width: fit-content;
   display: flex;
-  margin: 0 auto 30px;
+  margin: 30px auto 5px;
+  height: 60vh;
+}
+
+.button-addform {
+  background-color: #f2b807;
```

Comment:
I'm not sure but I believe your "Add item" is `disabled` for default. Maybe you can change your yellow to a soft gray.

![image](https://user-images.githubusercontent.com/8689897/142745804-2d502872-0c7a-4461-a48d-9944967b9ff7.png)

What do you think about it?

Reply:
Ready, it looks better like this